app/controllers/students_controller.rb

```diff
@@ -21,9 +21,10 @@ def update
   end
 
   def destroy
-    @user = User.find(params[:id])
+    @student = Student.find(params[:id])
```

Comment:
Instead of adding this line add the `:destroy` method to the before_action in line 3